templates/submission/list.html

```diff
@@ -67,6 +67,7 @@
             });
 
             // Draw the statistics graph.
+            var chart;
```

Comment:
Can we just initialize this to `null` instead of trying to pull nasty stuff to detect `undefined` down there?